test/index.js

```diff
@@ -90,24 +96,12 @@ describe('DoubleClick Floodlight', function(){
   describe('track', function(){
```

Comment:
@tsholmes I have a negative test for the `mapper`. Should I also have one where I actually call `test.track()`? 

But then what would be something similar to `didNotCall()`? Not sure if doing is sufficient?

```
test
   .track()
   .requests(0);
```